### PR TITLE
feat(datatable): add ElementCounter component

### PR DESCRIPTION
**Pull request by ajkl2533**
Closes FEP-1527

### CODE REVIEW

File: src/components/Datatable/ElementCounter/ElementCounter.types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface ElementCounterProps {
+  totalLength: number;
```
Comment (youssefssc) [inline src/components/Datatable/ElementCounter/ElementCounter.types.ts:2]: what about the case when the numbers are large? in the design for example the total is rendered as "1.05M" .

Comment (youssefssc) [reply src/components/Datatable/ElementCounter/ElementCounter.types.ts:2]: ok @ajkl2533 I see you are using `formatNumber` and it works but renders 1.05m with small "m"

Comment (PrincAm) [reply src/components/Datatable/ElementCounter/ElementCounter.types.ts:2]: maybe adding of this use case to storybook would be helpful

**Review by PrincAm — APPROVED** [review]
LGTM

File: src/components/Datatable/ElementCounter/ElementCounter.types.ts (hunk 1 above)
Comment (ajkl2533) [reply src/components/Datatable/ElementCounter/ElementCounter.types.ts:2]: sure guys :) fixed the uppercase and added abbreviated counts into SB

**Pull request merged**